src/org/sosy_lab/java_smt/solvers/yices2/package-info.java

```diff
@@ -0,0 +1,15 @@
+/*
+ *  JavaSMT is an API wrapper for a collection of SMT solvers.
+ *  This file is part of JavaSMT.
+ *
+ *  Copyright (C) 2007-2020  Dirk Beyer
+ *  All rights reserved.
+ *
+ *  SPDX-License-Identifier: Apache-2.0 AND GPL-3.0
```

Comment:
Is there a reason why this is a conjunction? This imposes the most restrictions on us, for example we are now always forced to honor both licenses in the future. If it were a disjunction, we could for example later drop the GPL if Yices moves to a different license.

Note that this would not change the licenses that users of the combination of JavaSMT would need to honor, because `Apache-2.0 AND GPL-3.0 AND (Apache-2.0 AND GPL-3.0)` (for JavaSMT + Yices + this package) is of course the same as `Apache-2.0 AND GPL-3.0 AND (Apache-2.0 OR GPL-3.0)`.